src/neuroglancer/sliceview/vector_graphics/backend.ts

```diff
@@ -35,21 +36,36 @@ export class PointChunk extends SliceViewChunk {
 
     let source = this.source;
 
-    this.systemMemoryBytes = source!.chunkBytes;
-    this.gpuMemoryBytes = source!.chunkBytes;
+    let chunkBytes: number = 0;
+    if (this.vertexPositions) {
+      chunkBytes = chunkBytes + this.vertexPositions!.buffer.byteLength; 
```

Comment:
I think you want just vertexPositions!.byteLength, not buffer.byteLength, in case it is just a view to a part of a larger buffer.  Also consider chunkBytes += .  Same applies to vertexNormals below.